CCGlucose/Classes/Glucose.swift

```diff
@@ -192,8 +192,14 @@ public class Glucose : NSObject {
     }
     
     func parseGlucoseMeasurement(data:NSData) {
-        let glucoseMeasurement = GlucoseMeasurement(data: data)
-        glucoseDelegate?.glucoseMeasurement(measurement: glucoseMeasurement)
+        // ensure the first byte is not zero before parsing the data
+        var values = [UInt8](repeating:0, count: data.length)
+        data.getBytes(&values, length: data.length)
```

Comment:
Instead of getting all the values, can you just do `let firstByte = data[0]`? (Maybe add `data.length > 0`)